src/app/screens/Survey/layout.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/prop-types */
```

Comment:
Delete this entire file and pass everything into the index. There's no need to separate both